### ISSUE TITLE
after 5000 iteration， the test image turn to black

### ISSUE THREAD

**Issue by sk284525908** [lengstrom/fast-style-transfer#40]
when i train a new style,  i use a test image, at first the style transfer doing well, but after 5000 iteration, all my transfer style image turn to black, after training, i use the checkpoint to style new image, it turn to black too, am i doing something wrong？

**Comment by lengstrom** [comment]
Please include the command and the data that you used if you want help!

**Comment by mdramos** [comment]
@sk284525908 update on this?

**Comment by mdramos** [comment]
seeing this on one of my tests, script is same as example

getting nan on my loss after first iteration

```
Epoch 0, Iteration: 1000, Loss: nan
```

**Comment by lengstrom** [comment]
I will look into this today.

**Comment by lengstrom** [comment]
I made some changes a few days ago that could have broken things.

**Comment by mdramos** [comment]
Sorry I think some of my dependencies, on this testing machine of mine, are outdated: 

Tensorflow: v1.0.0
Cuda: 5.1
Scipy: 0.17.0
Numpy: 1.13.1
PIL: 1.1.7

**Comment by mdramos** [comment]
Hi @lengstrom,

Actually, I reverted back to the commit [here](https://github.com/lengstrom/fast-style-transfer/commit/55809f4eafbe7d22632ac8fa254632ec41386d57) and everything went back to working. I tried early to update dependencies and debug a bit but the images after each checkpoint were just black.

**Comment by mdramos** [comment]
can create a new issue if youd like

**Comment by jeroenherczeg** [comment]
I have the same problem, all my test images are black

`python style.py --style GustavKlimt.jpg  --checkpoint-dir checkpoint --test jeroenherczeg.jpg --test-dir test --content-weight 1.5e1 --checkpoint-iterations 1000 --batch-size 10           
Train set has been trimmed slightly..
(1, 1238, 924, 3)
2017-08-04 15:58:25.203046: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:893] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2017-08-04 15:58:25.203262: I tensorflow/core/common_runtime/gpu/gpu_device.cc:955] Found device 0 with properties: 
name: GeForce GTX 1080
major: 6 minor: 1 memoryClockRate (GHz) 1.86
pciBusID 0000:01:00.0
Total memory: 7.92GiB
Free memory: 6.31GiB
2017-08-04 15:58:25.203285: I tensorflow/core/common_runtime/gpu/gpu_device.cc:976] DMA: 0 
2017-08-04 15:58:25.203289: I tensorflow/core/common_runtime/gpu/gpu_device.cc:986] 0:   Y 
2017-08-04 15:58:25.203293: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
2017-08-04 15:58:33.280587: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
UID: 59
^NEpoch 0, Iteration: 1000, Loss: nan
style: 3.02866e+23, content:8.57095e+06, tv: nan
2017-08-04 16:11:17.512847: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
Epoch 0, Iteration: 2000, Loss: nan
style: 3.02866e+23, content:7.45978e+06, tv: nan
2017-08-04 16:23:51.984912: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
Epoch 0, Iteration: 3000, Loss: nan
style: 3.02866e+23, content:7.89731e+06, tv: nan
2017-08-04 16:36:37.016055: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1045] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)`

**Comment by evan-wehi** [comment]
Getting the same problem. Evaluating produces a black images.
```
2017-08-05 05:25:50.187900: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-05 05:25:50.187940: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-05 05:25:50.187946: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-08-05 05:25:50.187951: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-08-05 05:25:50.187955: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2017-08-05 05:25:50.303321: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:893] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2017-08-05 05:25:50.303794: I tensorflow/core/common_runtime/gpu/gpu_device.cc:940] Found device 0 with properties:
name: Tesla K80
major: 3 minor: 7 memoryClockRate (GHz) 0.8235
pciBusID 0000:00:1e.0
Total memory: 11.17GiB
Free memory: 11.11GiB
2017-08-05 05:25:50.303831: I tensorflow/core/common_runtime/gpu/gpu_device.cc:961] DMA: 0
2017-08-05 05:25:50.303840: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971] 0:   Y
2017-08-05 05:25:50.303852: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla K80, pci bus id: 0000:00:1e.0)
2017-08-05 05:26:24.002351: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla K80, pci bus id: 0000:00:1e.0)
Train set has been trimmed slightly..
(1, 625, 1000, 3)
UID: 72
Epoch 0, Iteration: 2000, Loss: nan
style: 3.36575e+22, content:3.05473e+06, tv: nan
Epoch 0, Iteration: 4000, Loss: nan
style: 3.36575e+22, content:4.89044e+06, tv: nan
Epoch 0, Iteration: 6000, Loss: nan
style: 3.36575e+22, content:4.95108e+06, tv: nan
Epoch 0, Iteration: 8000, Loss: nan
style: 3.36575e+22, content:3.43088e+06, tv: nan
Epoch 0, Iteration: 10000, Loss: nan
style: 3.36575e+22, content:4.18378e+06, tv: nan
Epoch 0, Iteration: 12000, Loss: nan
style: 3.36575e+22, content:3.43025e+06, tv: nan
Epoch 0, Iteration: 14000, Loss: nan
style: 3.36575e+22, content:3.0429e+06, tv: nan
Epoch 0, Iteration: 16000, Loss: nan
style: 3.36575e+22, content:1.81728e+06, tv: nan
Epoch 0, Iteration: 18000, Loss: nan
style: 3.36575e+22, content:3.18618e+06, tv: nan
Epoch 0, Iteration: 20000, Loss: nan
style: 3.36575e+22, content:4.83422e+06, tv: nan
Epoch 1, Iteration: 2000, Loss: nan
style: 3.36575e+22, content:3.05473e+06, tv: nan
Epoch 1, Iteration: 4000, Loss: nan
style: 3.36575e+22, content:4.89044e+06, tv: nan
Epoch 1, Iteration: 6000, Loss: nan
style: 3.36575e+22, content:4.95108e+06, tv: nan
Epoch 1, Iteration: 8000, Loss: nan
style: 3.36575e+22, content:3.43088e+06, tv: nan
Epoch 1, Iteration: 10000, Loss: nan
style: 3.36575e+22, content:4.18378e+06, tv: nan
Epoch 1, Iteration: 12000, Loss: nan
style: 3.36575e+22, content:3.43025e+06, tv: nan
Epoch 1, Iteration: 14000, Loss: nan
style: 3.36575e+22, content:3.0429e+06, tv: nan
Epoch 1, Iteration: 16000, Loss: nan
style: 3.36575e+22, content:1.81728e+06, tv: nan
Epoch 1, Iteration: 18000, Loss: nan
style: 3.36575e+22, content:3.18618e+06, tv: nan
Epoch 1, Iteration: 20000, Loss: nan
style: 3.36575e+22, content:4.83422e+06, tv: nan
Epoch 1, Iteration: 20695, Loss: nan
style: 3.36575e+22, content:3.66257e+06, tv: nan
Training complete. For evaluation:
    `python evaluate.py --checkpoint styles_small/web_bridge.chkpt ...`
```

**Comment by mdramos** [comment]
for those who want to run this, the current head is broken.

revert to `55809f4eafbe7d22632ac8fa254632ec41386d57` for running
